README-config.md

```diff
@@ -14,6 +14,7 @@ Values that can be configured using environment variables:
 | `CIRCUIT_BREAKER_PRICE_CHANGE_THRESHOLD_MAX`    | No                              | It is the maximum the circuit breaker price threshold can scale up to. Any price change greater than this will assuredly open the circuit breaker.           | Provided as a ratio, e.g. 0.25 is 25%.                                                                                            | `0.25`                                                         | positive numbers                                                                              |
 | `CIRCUIT_BREAKER_PRICE_CHANGE_THRESHOLD_MIN`    | No                              | It is the base/minimum circuit breaker price change threshold. Every successful report resets the threshold to this value. If the difference between the new price to report and the most recently reported price exceeds this threshold, it may be reported, depending on how much time has elapsed. | Provided as a ratio, e.g. 0.25 is 25%.      | `0.15`      | positive numbers                                                                              |
 | `CIRCUIT_BREAKER_PRICE_CHANGE_THRESHOLD_TIME_MULTIPLIER`| No                      | It determines how quickly the calculated circuit breaker price change threshold scales in respect to time that has elapsed since the last report.            |                                                                                                                                   | `0.0075`                                                       | positive numbers                                                                              |
+| `CIRCUIT_BREAKER_DURATION_MS`                   | No                              | How long the oracle will stop reporting if the price moves more than the circuit breaker threshold.                                                          |                                                                                                                                   | `1200000` (20 minutes)                                         | positive integers                                                                             |
```

Comment:
Can we make the auto-recovery a setting? I imagine a couple of escenarios in which we'd recommend not to make it auto-recover

Reply:
oh I guess we could set it to a very large number if we wanted to do that.

Reply:
We can. What if we use negative duration values to mean that it shouldn't open?